spec/controllers/api/predictions_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Api::PredictionsController, type: :controller do
+  before(:each) do
+    controller.stub(:set_timezone)
+  end
+
+  describe 'index' do
+    before(:each) do
+      @prediction = build(:prediction)
+      @predictions = [@prediction]
+    end
+
+    context 'with valid API token' do
+      before(:each) do
+        @user = build(:user_with_email)
+        @user.stub(:api_token).and_return('token')
```

Comment:
Don't think you need to stub `api_token` here, just assign the token you want to use. E.g. `@user.api_token = 'token'`. It's better to avoid stubbing if possible because it exercises the actual code.